examples/chatmemberbot.py

```diff
@@ -80,7 +80,10 @@ async def track_chats(update: Update, context: ContextTypes.DEFAULT_TYPE) -> Non
     chat = update.effective_chat
     if chat.type == Chat.PRIVATE:
         if not was_member and is_member:
-            logger.info("%s started the bot", cause_name)
+            # This is not really needed in this particular case because /start command will run
```

Comment:
This comment is somewhat confusing IMO. If the user starts the bot, but then blocks & unblocks it, we do get the `my_chat_member` update about the unblocking and hence need this check, right?

Reply:
When I press "unblock" in my Telegram App, it unblocks the bot and automatically sends the `/start` command. The CommandHandler catches it, adds the user to the list and logs "user started the bot". 

So after unblocking I get two messages in the logger: user unblocked the bot and user started the bot.